src/extension/monitor/hibernate/src/main/resources/applicationContext-hib2.xml

```diff
@@ -1,27 +1,36 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd" default-autowire="byName">
  
-   <bean id="hibPropertyConfigurer" class="org.geoserver.config.GeoServerPropertyConfigurer"
-        depends-on="geoServerLoader">
-      <constructor-arg ref="dataDirectory"/>
-      <property name="location" value="file:monitoring/hibernate.properties"/>
-      <property name="properties">
-        <props>
-          <prop key="database">H2</prop>
-          <prop key="databasePlatform">org.geoserver.monitor.hib.H2Dialect</prop>
-          <prop key="showSql">false</prop>
-          <prop key="generateDdl">true</prop>
-          
-          <prop key="hibernate.hbm2ddl.auto">update</prop>
-          <prop key="hibernate.generate_statistics">true</prop>
-          <prop key="hibernate.session_factory_name">SessionFactory</prop>
-          <prop key="hibernate.bytecode.use_reflection_optimizer">true</prop>
-          <prop key="hibernate.show_sql">false</prop>
-          <prop key="hibernate.use_sql_comments">true</prop>
-          <prop key="hibernate.format_sql">true</prop>
-          <!--prop key="hibernate.jdbc.use_streams_for_binary">true</prop-->
-        </props>
-      </property>
+   <bean id="monitoring-hibPropertyConfigurer" class="org.geoserver.config.GeoServerPropertyConfigurer">
+
+        <property name="order" value="100"/>
+        
+        <constructor-arg ref="dataDirectory"/>
+
+        <property name="ignoreResourceNotFound" value="true" />
+        <property name="ignoreUnresolvablePlaceholders" value="true" />
```

Comment:
I was thinking exactly of these properties to solve the GWC property configurer nuke... did you already try it by any chance?